dashboard/src/components/AppView/AppControls/AppControls.tsx

```diff
@@ -47,7 +45,7 @@ class AppControls extends React.Component<IAppControlsProps, IAppControlsState>
         {/* If the app has been deleted hide the upgrade button */}
         {!deleted && (
           <UpgradeButton
-            updateVersion={(updateInfo && updateInfo.latestVersion) || ""}
+            updateVersion={(app.updateInfo && app.updateInfo.latestVersion) || ""}
```

Comment:
empty string? why not undefined? `UpgradeButton` can handle whether it is set or not